platforms/cpuinfo.go

```diff
@@ -129,3 +136,25 @@ func getCPUVariant() string {
 
 	return variant
 }
+
+// getCPUArchitecture from uname -m
+// when getCPUInfo func from /proc/cpuinfo read error,
```

Comment:
What error?

Reply:
err:`getCPUInfo for pattern: Cpu architecture: not found`
ref: https://github.com/containerd/containerd/issues/4109

Reply:
We shouldn't document the implementation of getCPUVariant here.

Reply:
Do you mean remove 141 line,just keep 140 line.

Reply:
Do you mean remove 141 line,just keep 140 line.